packages/fscomponents/src/types/ModalHalfScreen.ts

```diff
@@ -0,0 +1,24 @@
+export type AccessibilityTraits =
```

Comment:
The linter is complaining because the strings should be in single quotes.

Also, can you please rename this file to something like AccessibilityTypes? I suspect we'll want to use these types in additional places beyond just ModalHalfScreen.

Reply:
Fixed the linter issue. I didn't change the name since they are the exact names for the properties (accessibilityTraits for iOS and accessibilityComponentType for Android). However, I put the two types in Store.ts so all the components can use the types.